devdoc/umock_c_negative_tests_requirements.md

```diff
@@ -90,6 +91,19 @@ void umock_c_negative_tests_fail_call(size_t index);
 
 ## umock_c_negative_tests_call_count
 
+## umock_c_negative_tests_get_fail_call
```

Comment:
>umock_c_negative_tests_get_fail_call [](start = 3, length = 36)

Can you please add with a short description in the uber umock doc? /doc/umock_c.md